<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FusionFields</class>
 <widget class="QWidget" name="FusionFields">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="4">
    <widget class="ChallengeTask" name="challengeTask" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>9</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Button" name="btnRestart" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="GameField" name="gameField" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>6</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="ItemBoard" name="itemBoard" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="Board" name="board" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="GameEndWidget" name="gameEndWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>9</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="Button" name="btnMenu" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="Notification" name="notification" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="Button" name="btnNav" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Button</class>
   <extends>QWidget</extends>
   <header>button.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameField</class>
   <extends>QWidget</extends>
   <header>gamefield.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Board</class>
   <extends>QWidget</extends>
   <header>board.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Notification</class>
   <extends>QWidget</extends>
   <header>notification.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ItemBoard</class>
   <extends>QWidget</extends>
   <header>itemboard.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChallengeTask</class>
   <extends>QWidget</extends>
   <header>challengetask.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameEndWidget</class>
   <extends>QWidget</extends>
   <header>gameendwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
